connexion/cli.py

```diff
@@ -1,6 +1,7 @@
 import importlib
 import logging
 import sys
+from itertools import chain
```

Comment:
Redundant import?

Reply:
I already remove this line, this code is outdated